src/registry/local.rs

```diff
@@ -105,6 +105,7 @@ mod tests {
     use tokio::fs;
 
     #[tokio::test]
+    #[ignore = "gzip header is invalid"]
```

Comment:
Because the recomputed gzip header breaks depending on the system (i guess due to different gzip versions present or other system settings)

We should compare file contents, not hash i guess